frontend/projects/setup-wizard/src/app/pages/transfer/transfer.page.ts

```diff
@@ -0,0 +1,74 @@
+import { Component } from '@angular/core'
```

Comment:
This component seems very similar to others. Maybe there is an abstraction here.

Reply:
There probably is a base component that could be abstracted, which could be extended with different `getDrive`, and `select`/`chooseDrive` implementations, between the embassy, recover, and transfer pages. They are distinct enough that I decided to keep them separate for now, mainly for clarity.

I think the entire approach here is roundabout and repetitive, and think an overall refactor would be ideal, but that is out of scope for this task.